Monika After Story/game/zz_calendar.rpy

```diff
@@ -0,0 +1,1693 @@
+#           [3]: True if we want the button bold, False if not
+#           [4]: integer spacing between this button and the regular buttons
+#               NOTE: must be >= 0
+#       (Default: None)
+#   final_item - represents the final (usually quit item) of the menu
+#       tuple of the following format:
+#           [0]: text of the button
+#           [1]: return value of the button
+#           [2]: True if we want the button italics, False if not
+#           [3]: True if we want the button bold, False if not
+#           [4]: integer spacing between this button and the regular buttons
+#               NOTE: must be >= 0
+#       (Default: None)
+#   mask - hex color that will be used for the mask that will cover the screen
+#       if None there won't be any mask
+#   frame - route to the image used as backround for the list
+screen mas_calendar_events_scrollable_list(items, display_area, scroll_align, first_item=None, final_item=None, mask="#000000B2", frame="mod_assets/calendar/calendar_bg.png"):
+        style_prefix "scrollable_menu"
+
+        zorder 51
+
+        if mask:
+            add Solid(mask)
+
+        fixed:
+            area display_area
+            if frame:
+                add Frame(frame, 60, 60)
+
+            bar adjustment prev_adj style "vscrollbar" xalign scroll_align
+
+            viewport:
+                yadjustment prev_adj
+                mousewheel True
+
+                vbox:
+
+                    if first_item:
+
+                        text _(first_item[0]):
+                            if first_item[1]:
+                                italic True
+                            if first_item[2]:
+                                bold True
+                            xpos 0.2
+                            ypos 0.5
+
+                    null height 30
+
+
+                    for item_prompt,is_italic,is_bold in items:
+                        text item_prompt:
+                            if is_italic:
+                                italic True
+                            if is_bold:
+                                bold True
+                            xpos 0.05
+
+
+                    if final_item:
+                        if final_item[4] > 0:
+                            null height final_item[4]
+
+                        textbutton _(final_item[0]):
+                            if final_item[2]:
+                                text_italic True
+                            if final_item[3]:
+                                text_bold True
+                            background None
+                            hover_sound gui.hover_sound
+                            activate_sound gui.activate_sound
+
+                            action Return(final_item[1])
+
+
+label _first_time_calendar_use:
+    m 1eub "Oh, I see you noticed that pretty calendar hanging in the wall, [player]"
+    m "It helps me to keep track of important events, ehehe~"
```

Comment:
+    m 1eub "Oh, I see you noticed that pretty calendar hanging in the wall, [player]"
+    m "It helps me to keep track of important events, ehehe~" ``

I'd change 'hanging in' to "hanging on" and change "helps me to keep" to "helps me keep".